subsys/suit/stream/Kconfig

```diff
@@ -81,6 +82,7 @@ menuconfig SUIT_STREAM_SINK_SDFW_RECOVERY
 	bool "Enable SDFW Recovery sink"
 	depends on SUIT_UTILS
 	depends on SOC_SERIES_NRF54HX
+	depends on SOC_SERIES_NRF92X
```

Comment:
I doubt that you can enable both `SOC_SERIES_NRF54HX` and `SOC_SERIES_NRF92X` at the same time..

Reply:
Fixed.

Reply:
Fixed.